mechanics/CREATING-A-SANDBOX-REPO.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Creating a sandbox repo"
+linkTitle: "Sandbox repo process"
+weight: 30
+type: "docs"
+---
+
+<!-- NOTE: some portion of this document is also reproduced in
+/.github/ISSUE_TEMPLATES/new-repo.md. If you are updating the steps here, you
+may need to look there, too. -->
+
+# Background
+
+The [`knative-sandbox`](https://github.com/knative-sandbox) GitHub org exists to
+hold "non-core" Knative components which are owned and sponsored by a Working
+Group. See the [Knative Repository Guidelines](../REPOSITORY-GUIDELINES.md) for
+more details on the requirements for the `knative-sandbox` org.
+
+## Criteria
+
+A Working Group Lead (either
+[technical](../ROLES.md#working-group-technical-lead) or
+[execution](../ROLES.md#working-group-execution-lead)) may request a new repo in
+`knative-sandbox` by filing an issue in the
+[knative/community](https://github.com/knative/community/issues/new?template=new-repo.md)
+repo. Once filed, the TOC should handle these promptly, though it should also be
+considered fine to ping members or the group on Slack if it hasn't been acted on
+in a few days. Generally, the request will be granted, though the TOC may have
+additional questions or suggest an alternate mechanism in some cases.
+
+Some of the following steps may require permissions that are only available to
+the TOC or Steering Committee, though others are largely self-service or require
+other WGs to review and approve impacting changes.
+
+### Technical requirements
+
+- Any contributed code should be contributed by the original authors or
+  copyright holders under an Apache 2.0 license. See also
+  [this section of the repository guidelines](../REPOSITORY-GUIDELINES.md#creation).
+
+- Names between `knative-sandbox` and the main `knative` repo should not
+  overlap. This simplifies promoting repos between the two orgs and setting up
+  `knative.dev` import paths for golang.
+
+- Prow automation for tests is encouraged but not required for
+  `knative-sandbox`, but the Google CLA bot and OWNERS files/tide merge should
+  be enforced.
+
+## Process (to be executed by TOC or Steering member)
+
+1. (Requires Org owner) Create the new repo in
+   https://github.com/knative-sandbox using the "New" button. Set the repo to
```

Comment:
@tcnghia and I learned the hard way yesterday that peribolos can create skeleton repos for you, so unless we're starting from a template repo, that might be a useful trick to incorporate.

Reply:
I think a set of template repos is probably the way to go.

base --> controller
  \--> source
  \--> ???

But I'm worried about building more than we need, so let's start with the sample-controller and sample-source, plus a barebones one, and see where we get.